### ISSUE TITLE
Shaking up the teller appears only after completing Postman Pat

### ISSUE THREAD

**Issue by tperamaki** [TarkovTracker/TarkovTracker#354]
Shaking up the teller does not appear when it should, but instead seems to require Postman Pat to be completed (id 7), but the data says that it requires only quest with id 6, which would be correct.

**Comment by Shebuka** [comment]
**Shaking up the teller** should appear after you completed **Ice Cream Cones** AND got **LL2 with Prapor**. Is this not correct?

**Comment by tperamaki** [comment]
Does tarkovTracker track the LL somehow? Because I got the LL2 through dailyes and such, and this quest just didn't show up on tarkovTracker, but in the game it did

**Comment by Shebuka** [comment]
Right now there is no way to track LL and adjust it for dailies... we need to wait for @thaddeus to implement this.

**Comment by DysektAI** [comment]
Is this still an issue or has this been resolved?

**Comment by DysektAI** [comment]
## Has your issue been resolved? @tperamaki 
- **Yes** → please close this issue (use **Close issue** or **Close with comment** below).
- **No** → leave a short comment so others know it's still a problem.

> [!IMPORTANT]
> This repository and [TarkovTracker.io](https://TarkovTracker.io) are **currently unmaintained**.  
> It’s unlikely that new issues or pull requests will receive a response from the original maintainers.

### 👉 Looking for the active project?
- **Website:** [TarkovTracker.org](https://TarkovTracker.org)
- **Source code:** [tarkovtracker-org/TarkovTracker](https://github.com/tarkovtracker-org/TarkovTracker)
- **Open a new issue here:** https://github.com/tarkovtracker-org/TarkovTracker/issues/new/choose
- **Project status & changelog:** #455 

_Thanks for keeping things tidy and for using TarkovTracker!_
